config/cops.yml

```diff
@@ -0,0 +1,246 @@
+Style/AccessModifierIndentation: 200_000
```

Comment:
I just assumed that these would be tweaked in the near future so I added them in real quick.

If we want to only include cops that we change I can remove all of these for now.